leveldb-library.podspec

```diff
@@ -1,6 +1,10 @@
+# CocoaPods 1.x currently does not support libraries with C++ headers. See
+# https://github.com/CocoaPods/CocoaPods/issues/5152. The workaround is to use
+# the 1.2.1.beta.1 with its new option --skip-import-validation.
```

Comment:
Details at https://github.com/firebase/leveldb/pull/1/files. Would it be helpful to add comparable instructions to this PR?